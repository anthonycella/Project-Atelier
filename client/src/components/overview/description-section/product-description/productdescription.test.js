import renderer from 'react-test-renderer';
import ProductDescription from './productdescription.jsx';

it('Product Description renders to the page', () => {
  const colorScheme = { background: 'black' };
  const data = {
    catchphrase: 'catchphrase',
    description: 'one ring to rule them all',
  };
  const component = renderer.create(
    <ProductDescription colorScheme={colorScheme} data={data} />
  )
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
