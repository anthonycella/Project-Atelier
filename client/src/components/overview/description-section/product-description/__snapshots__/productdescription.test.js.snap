// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product Description renders to the page 1`] = `
<div
  className="sc-dkzDqf jGAVXb"
>
  <div
    className="sc-bczRLJ dqmtFn"
  >
    catchphrase
  </div>
  <div
    className="sc-gsnTZi dSBnGn"
  >
    one ring to rule them all
  </div>
</div>
`;
